tests/datasource/data_connector/test_configured_asset_azure_data_connector.py

```diff
@@ -33,7 +33,7 @@
 
 @pytest.fixture
 def expected_config_dict():
-    """Used to validate `self_check()` and `test_yaml_config()` outputs."""
```

Comment:
I'm pretty sure everything related to `self_check()`can also be removed at some point, but that's for another PR. It was all part of this `test_yaml_config()` workflow.

Reply:
Thanks. I had a hunch, but wanted to limit scope!